typings/single-spa.d.ts

```diff
@@ -120,9 +120,12 @@ declare module "single-spa" {
     warningMillis?: number
   ): void;
 
+  import System from "systemjs";
   type Application<ExtraProps = {}> =
     | LifeCycles<ExtraProps>
-    | ((config: ExtraProps & AppProps) => Promise<LifeCycles<ExtraProps>>);
+    | ((
+        config: ExtraProps & AppProps
+      ) => Promise<LifeCycles<ExtraProps>> | Promise<System.Module>);
```

Comment:
`System.Module` is a less specific type than `Lifecycles<ExtraProps>`. Why is this desirable? Does typescript complain when it sees a `System.import()` for the load function?